guidance/README.adoc

```diff
@@ -0,0 +1,31 @@
+= Guiding principles for Automation Good Practices
+
+As an overall guiding principle for designing good automation, the Zen of Python, by Tim Peters, serves as a good guidepost to follow. The principles defined in this are very applicable and can give guidance when the better practice is  unclear.
+
```

Comment:
Aw, no instructions to run "import this" in a python terminal? :) Looks good though!

Reply:
Need to check for plug-in compatibility, review and check back next meeting

Reply:
It comes a bit like a hair on the soup (French saying, "out of the blue") at this place. I would suggest to rename the "structures" section to "structures and principles", and add this recommendation there (freely as first recommendation), under a title like "Follow the Zen of Python for your automation" (a better title is surely possible).